src/main/java/nextstep/courses/entity/SessionEntity.java

```diff
@@ -0,0 +1,91 @@
+import java.time.LocalDateTime;
+
+import static nextstep.courses.domain.session.Session.NOT_ASSIGNED;
+
+@Getter
+public class SessionEntity {
+
+    private final Long id;
+    private final String coverFilePath;
+    private final SessionState sessionState;
+    private final int enrollment;
+    private final int maxEnrollment;
+    private final long sessionFee;
+    private final LocalDateTime startDate;
+    private final LocalDateTime endDate;
+
+    public SessionEntity(Long id, String coverFilePath, SessionState sessionState,
+                         int enrollment, int maxEnrollment, long sessionFee,
+                         Timestamp startDate, Timestamp endDate) {
+        this(id, coverFilePath, sessionState, enrollment, maxEnrollment,
+                sessionFee, toLocalDateTime(startDate), toLocalDateTime(endDate));
+    }
+
+    public SessionEntity(String coverFilePath, SessionState sessionState,
+                         int enrollment, int maxEnrollment, long sessionFee,
+                         Timestamp startDate, Timestamp endDate) {
+        this(null, coverFilePath, sessionState, enrollment, maxEnrollment,
+                sessionFee, toLocalDateTime(startDate), toLocalDateTime(endDate));
+    }
+
+    public SessionEntity(String coverFilePath, SessionState sessionState,
+                         int enrollment, int maxEnrollment, long sessionFee,
+                         LocalDateTime startDate, LocalDateTime endDate) {
+        this(null, coverFilePath, sessionState, enrollment, maxEnrollment, sessionFee, startDate, endDate);
+    }
+
+    public SessionEntity(Long id, String coverFilePath, SessionState sessionState,
+                         int enrollment, int maxEnrollment, long sessionFee,
+                         LocalDateTime startDate, LocalDateTime endDate) {
+        this.id = id;
+        this.coverFilePath = coverFilePath;
+        this.sessionState = sessionState;
+        this.enrollment = enrollment;
+        this.maxEnrollment = maxEnrollment;
+        this.sessionFee = sessionFee;
+        this.startDate = startDate;
+        this.endDate = endDate;
+    }
+
+    private static LocalDateTime toLocalDateTime(Timestamp timestamp) {
+        if (timestamp == null) {
+            return null;
+        }
+        return timestamp.toLocalDateTime();
+    }
+
+    public Session toDomain() {
+        SessionBuilder sessionBuilder = SessionBuilder.builder()
+                .id(id)
+                .sessionState(sessionState)
+                .coverImage(coverFilePath)
+                .enrollment(enrollment)
+                .maxEnrollment(maxEnrollment)
+                .startDate(startDate)
+                .endDate(endDate);
+
+        if (sessionFee == 0) {
+            return sessionBuilder.sessionType(SessionType.FREE).build();
+        }
+        return sessionBuilder.sessionType(SessionType.PAID).sessionFee(sessionFee).build();
+    }
+
+    public static SessionEntity toEntity(Session session) {
```

Comment:
CourseEntity.toEntity(course) 보단 CourseEntity.from(course) 가 의미전달이 잘될것 같아요 :)

```suggestion
    public static SessionEntity from(Session session) {
```